bucket/qgroundcontrol.json

```diff
@@ -0,0 +1,19 @@
+{
+    "homepage": "http://qgroundcontrol.com",
+    "description": "Cross-platform ground control station for drones.",
+    "license": "Apache-2.0|GPL-3.0-only",
+    "version": "v4.0.6",
+    "url": "https://github.com/mavlink/qgroundcontrol/releases/download/v4.0.6/QGroundControl-installer.exe#/dl.7z",
+    "hash": "76dd7acaf6c5d6d3de255d88fda0197a29aab00ab1f62d90d085e81d4fe8daf3",
+    "bin": "QGroundControl-installer.exe",
+    "shortcuts": [
+        [
+            "qgroundcontrol.exe",
+            "QGroundControl"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/mavlink/qgroundcontrol/releases/download/$version/QGroundControl-installer.exe"
+    }
```

Comment:
```suggestion
    "version": "4.0.6",
    "description": "Intuitive and Powerful Ground Control Station for the MAVLink protocol",
    "homepage": "http://qgroundcontrol.com",
    "license": "Apache-2.0|GPL-3.0-only",
    "suggest": {
        "vcredist": "extras/vcredist2019"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/mavlink/qgroundcontrol/releases/download/v4.0.6/QGroundControl-installer.exe#/dl.7z",
            "hash": "76dd7acaf6c5d6d3de255d88fda0197a29aab00ab1f62d90d085e81d4fe8daf3"
        }
    },
    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\vc_red*\", \"$dir\\*uninsta*exe\" -Recurse",
    "bin": "QGroundControl.exe",
    "shortcuts": [
        [
            "QGroundControl.exe",
            "QGroundControl"
        ]
    ],
    "checkver": {
        "github": "https://github.com/mavlink/qgroundcontrol"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/mavlink/qgroundcontrol/releases/download/v$version/QGroundControl-installer.exe#/dl.7z"
            }
        }
    }
```

Reply:
That's what's on the website:
![Screenshot_2020 05 07_00h41m28s_001_](https://user-images.githubusercontent.com/1368405/81235587-9bb33e00-8ffb-11ea-9c65-5a9ccd996064.png)

Source: http://qgroundcontrol.com/